babel/core.py

```diff
@@ -180,6 +181,7 @@ def __init__(
         :param territory: the territory (country or region) code
         :param script: the script code
         :param variant: the variant code
+        :param modifier: a modifier ('@variant')
```

Comment:
Does this imply the modifier has the `@` intact here? It shouldn't, IMO..?

Reply:
I agree. I was studying the docs, and often saw '@variant', which I thought would be read as "at-variant", and so I included it. But it should probably go. Done.